packages/labextension/package.json

```diff
@@ -80,13 +80,14 @@
     },
     "outputDir": "../../nbdime/labextension",
     "sharedPackages": {
-      "codemirror": {
```

Comment:
I thought this helped reducing the bundle size (as CM would not be needed in the lab bundle). Did I misunderstand? I never did get a full understanding of all the aspects of this setting.

Reply:
The shared packages are pre-populated by JupyterLab and some of the core dependencies (through the builder) - this includes the new `@codemirror/` packages as some have requirements to be singleton to work.

So in summary, we can drop codemirror here it is implicitly done for us by JupyterLab.

Reply:
Thanks for the update 👍